ScoutSuite/providers/aws/facade/cloudformation.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from ScoutSuite.providers.aws.facade.utils import AWSFacadeUtils
+from ScoutSuite.providers.utils import run_concurrently
+
+
+class CloudFormation:
+    async def get_stacks(self, region: str):
+        stacks = await AWSFacadeUtils.get_all_pages('cloudformation', region, 'list_stacks', 'StackSummaries')
+
+        client = await AWSFacadeUtils.get_client('cloudformation', region)
```

Comment:
```suggestion
        client = AWSFacadeUtils.get_client('cloudformation', region)
```

Sorry about that, we made `get_client` non async yesterday in the end, haha.